honeybee_grasshopper_radiance/src/HB Extract HDR.py

```diff
@@ -0,0 +1,218 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2021, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Interpolate or extrapolate a High Dynamic Range (HDR) image file from another
+HDR image file.
+_
+Recommended use is to extract 180 FOV (-vh 180 -vv 180) angular or hemispherical
+HDR images from a 360 FOV (-vh 360 -vv 360) angular HDR image. Alternatively
+conversions between 180 FOV angular and hemispherical HDR images can be made.
+-
+
+    Args:
+        _view: A view to interpolate or extrapolate. Use "HB View" to create a
+            view.
+        _resolution_: An integer for the dimension of the output image in pixels.
+            If extracting a 180 FOV angular or hemispherical HDR image from a
+            360 FOV HDR image, the default resolution is 1/3 of the resolution of
+            _hdr. If converting between 180 FOV angular or hemispherical HDR 
+            images, the default resolution is that of _hdr.
+        _hdr: Path to a High Dynamic Range (HDR) image file from which to
+            interpolate or extrapolate a view.
```

Comment:
I usually try to put the required inputs towards the top of the component. It seems a little unconventional to have one at the bottom that isn't a boolean toggle for `_run`.